### PR TITLE
Fixes #11727: Rudder installation fails on SLES12

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/11727

### CODE REVIEW

File: rudder-agent/SPECS/rudder-agent.spec [hunk 1]
```diff
@@ -350,6 +350,10 @@ then
 %endif
   CFRUDDER_FIRST_INSTALL=1
 fi
+# mandatory with systemd wrapper for old init
+%if 0%{?suse_version} && 0%{?suse_version} >= 1315
```
Comment (peckpeck) [inline rudder-agent/SPECS/rudder-agent.spec]: This is sles12 and later, @amousset dou you think the condition should be larger ?

Comment (ncharles) [reply rudder-agent/SPECS/rudder-agent.spec]: isn't it something we should do on other distro as well (debian9, rhel 7) ?

Comment (peckpeck) [reply rudder-agent/SPECS/rudder-agent.spec]: It it tied to how chkconfig --add and update-rc.d is done by the distro (this command should reload by itself). So this is distro specific and it has not been reported on other distro yet.
That's why i'm asking for comment from @amousset

File: rudder-reports/SPECS/rudder-reports.spec [hunk 2]
```diff
@@ -181,6 +181,9 @@ chkconfig --add ${POSTGRESQL_SERVICE_NAME} >/dev/null 2>&1
   chkconfig ${POSTGRESQL_SERVICE_NAME} on >/dev/null 2>&1
 %endif
 echo " Done"
+# mandatory with systemd wrapper for old init
+%if 0%{?suse_version} && 0%{?suse_version} >= 1315
+systemctl daemon-reload
```
Comment (amousset) [inline rudder-reports/SPECS/rudder-reports.spec]: missing `%endif`

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**